pkgs/development/ocaml-modules/index/default.nix

```diff
@@ -1,34 +1,49 @@
-{ lib, fetchurl, buildDunePackage
+{ lib, fetchFromGitHub, buildDunePackage
 , repr, ppx_repr, fmt, logs, mtime, stdlib-shims
 , cmdliner, progress, semaphore-compat
 , alcotest, crowbar, re
 }:
 
 buildDunePackage rec {
   pname = "index";
-  version = "1.3.0";
+  version = "1.3.1";
 
-  minimumOCamlVersion = "4.08";
+  src = fetchFromGitHub {
```

Comment:
Please use `fetchurl` and the URL that was there before.

Reply:
Done.